packages/grafana-ui/src/utils/storybook/withStoryContainer.tsx

```diff
@@ -43,38 +43,12 @@ const StoryContainer: React.FC<{ width?: number; height?: number; showBoundaries
   );
 };
 
-export const withStoryContainer = (story: RenderFunction) => {
-  const CONTAINER_GROUP = 'Container options';
-  // ---
-  const containerBoundary = boolean('Show container boundary', false, CONTAINER_GROUP);
-  const fullWidthContainer = boolean('Full width container', false, CONTAINER_GROUP);
-  const containerWidth = number(
-    'Container width',
-    300,
-    {
-      range: true,
-      min: 100,
-      max: 500,
-      step: 10,
-    },
-    CONTAINER_GROUP
-  );
-  const containerHeight = number(
-    'Container height',
-    0,
-    {
-      range: true,
-      min: 100,
-      max: 500,
-      step: 10,
-    },
-    CONTAINER_GROUP
-  );
+export const withStoryContainer = (story: RenderFunction, context: StoryContext) => {
   return (
     <StoryContainer
-      width={fullWidthContainer ? undefined : containerWidth}
-      height={containerHeight}
-      showBoundaries={containerBoundary}
+      width={context.args.containerWidth}
```

Comment:
width prop isn't working because you're using lower case `containerWidth` here

Reply:
Great catch! pushing a fix right away